platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -247,34 +214,31 @@ There are no new privacy concerns with the auto-deploy as user data is not invol
 
 #### Risks
 
-- Newly added content from CMS without associated transformer schemas could fail the build.
-  - New content introduces the possibility of new discrepancies, which would fail any validations that depend on diffs between the GraphQL and CMS export builds.
-  - Having more content will inflate the runtime of the content-only deploy and could potentially exceed the scheduled interval.
-  - The best defense against this is accounting for null and empty fields in transformers.
+- Having more content will inflate the runtime of the content-only deploy and could potentially exceed the scheduled interval.
 
```

Comment:
- Conflicts with the full build. We need to test that running the content only build concurrently with the full build is handled correctly no matter which one is the first.

Reply:
@drorva We did look into that at one point, is this what you're referencing? https://github.com/department-of-veterans-affairs/va.gov-team/issues/18393